anon-aadhaar-config.toml

```diff
@@ -19,3 +19,9 @@ name = "aadhaar-verifier"                      # Name of the circuit
 # See https://github.com/oskarth/mopro/pull/37 and https://github.com/oskarth/mopro/pull/38
 use_dylib = false          # Options: true, false
 name = "anonAadhaar" # Name of the dylib file, only used if use_dylib is true
+
+[witness]
+# Note: circom-witness-rs is experimental
+# See https://github.com/oskarth/mopro/issues/32 for updates
+# Only works for keccak256_256_test circuit now
+use_witness = false # Options: true, false
```

Comment:
I still find this variable name confusing, we are always using a witness

A more descriptive name would be `use_native_witness_generation` or something like this